clients/client-topia/src/factories/UserFactory.ts

```diff
@@ -8,8 +8,14 @@ export class UserFactory {
     this.topia = topia;
   }
 
-  create(email: string, options?: UserOptionalInterface): User {
-    return new User(this.topia, email, options);
+  create(options?: UserOptionalInterface): User {
```

Comment:
I don't view the visitorId and the urlSlug as optional here. They are mandatory for a visitor to exist.

I suppose we need to start thinking about "visitors" and "users"

eventually we'll want some uniquely identifiable ID for a user that is not in a world at the moment. That isn't a visitor. It's a platform user. And a UID is a mandatory characteristic of that.

Perhaps this User, UserFactory, etc. should be renamed "Visitor"? World classes have "visitor" attributes?

Reply:
Can we chat about this in the morning?

Reply:
This would also more accurately reflect what our public API endpoints are designed to do w/ "visitors" and "users" by email.